meters.csv

```diff
@@ -47,6 +47,7 @@ http://fcc.io/EWQ100T,Itron,100T,Gas,9,903,926.9
 ,Itron,40GB,Gas,2,,
 http://fcc.io/EWQ100W,Itron,100W,Water,11,903,927
 ,Itron,100WP,Water,11,,
+,Itron,100WR,Water,171,,
```

Comment:
SCM+ messages include extra  information in the EndpointType field. The 4 least significant bits of this field is the same as is found in the SCM messages. Probably should record 11 (0x0B) here instead.

Reply:
sure thing, that makes sense.  Appreciate the review and help.  I submitted an update.